### ISSUE TITLE
Prompt validators not running

### ISSUE THREAD

**Issue by Ron537** [yibn2008/mock-inquirer#2]
### Current behavior
Mocking answers returns the mocked data without running validations before

### Expected behavior
`mock-inquirer` should run validators and throw error when mocked answer is not valid

### Reproduction
```javascript
const mockInquirer = require('mock-inquirer')
const inquirer = require('inquirer')
const co = require('co')

// start mock
co(function * () {
  let reset = mockInquirer([{
    number: 'hello'
  }, {}])

  let anwser = yield inquirer.prompt([{
    type: 'input',
    message: 'Enter number',
    name: 'number',
    validate: function(input) {
            if(/\d+/.test(input)) return true;
            return 'Invalid number entered';
    }
  }]);

  // print anwser:
  // { number: 'hello' }
  console.log(anwsers)

  // reset mock
  reset()
}).catch(err => {
  console.error(err)
})
```

**Comment by Ron537** [comment]
Submitted a pull request to solve this issue..

**Comment by yibn2008** [comment]
@Ron537 nice work! new version released: + mock-inquirer@1.0.2
